data/constants/playlist.py

```diff
@@ -0,0 +1,28 @@
+playlists = {
```

Comment:
Why do we need this?
Can't we use the enum?

Reply:
Because the enum doesn't have the strings that are used in the frontend. I mean we can, but I'm not sure its worth it because this currently isn't used anywhere.